README.md

```diff
@@ -7,65 +7,46 @@
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=cego_gitte&metric=coverage)](https://sonarcloud.io/dashboard?id=cego_gitte)
 [![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=cego_gitte&metric=code_smells)](https://sonarcloud.io/dashboard?id=cego_gitte)
 
+Tool to help keep a range of projects up to date with git version control, and also help execute commands and scripts across projects. For configuration options please refer to [config documentation](./docs/config.md).
+
 ## Installation
 
-### Linux
```

Comment:
Don't we need linux instructions anymore? @LauJosefsen

Reply:
I readded linux setup.